website/docs/components/code-block/partials/code/how-to-use.md

```diff
@@ -0,0 +1,148 @@
+## How to use this component
+
+The basic invocation requires `@value` to be passed.
+
+```handlebars
+<Hds::CodeBlock
+  @value="aws ec2 --region us-west-1 accept-vpc-peering-connection"
+/>
+```
+
+### Title and description
+
+Optionally, you can pass a title or a description.
+
+```handlebars
+<Hds::CodeBlock
+  @language="bash"
+  @value="aws ec2 --region us-west-1 accept-vpc-peering-connection"
+as |CB|>
+  <CB.Title>
+    Acceptance instructions
+  </CB.Title>
+</Hds::CodeBlock>
+```
+
+```handlebars
+<Hds::CodeBlock
+  @language="bash"
+  @value="export VAULT_ADDR=https://0242ac170030.aws.hcp.dev:8200;
+export VAULT_NAMESPACE=admin"
+as |CB|>
+  <CB.Description>
+    Export your cluster’s public URL and the default <a>namespace</a> called admin.
+  </CB.Description>
+</Hds::CodeBlock>
+```
+
+### Language
+
+The `language` argument sets the syntax highlighting used.
+
+```handlebars
+<Hds::CodeBlock
+  @language="go"
+  @value="package main
+import fmt
+func main() {
+  fmt.Println(helloWorld)
+}"
+/>
+```
+
+### Copy button
+
+Set `hasCopyButton` to `true` to display a button for users to copy `CodeBlock` content to their computer clipboard.
+
+```handlebars
+<Hds::CodeBlock
+  @language="javascript"
+  @hasCopyButton={{true}}
+  @value="let codeLang=`JavaScript`;
+console.log(`I am ${codeLang} code`);"
+/>
+```
+
+### Line numbers
+
+Line numbers are displayed by default. Set `hasLineNumbers` to `false` to hide them.
+
+```handlebars
+<Hds::CodeBlock
+  @language="javascript"
+  @hasLineNumbers={{false}}
+  @value="let codeLang=`JavaScript`;
+console.log(`I am ${codeLang} code`);"
+/>
+```
+
+### Line wrapping
+
+By default, long lines of code will overflow the `CodeBlock` container requiring users to scroll to view the full content. Setting `hasLineWrapping` to `true` will wrap long lines of code instead.
+
+```handlebars
+<Hds::CodeBlock
+  @language="javascript"
+  @hasLineWrapping={{true}}
+  @value="console.log(`I am JavaScript code`, `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam`);"
+/>
+```
+
+### Highlight lines
+
+Highlight either individual code lines or a range of code lines. (Examples: "2, 4", "6-10")
+
+```handlebars
```

Comment:
In this example, the highlighting doesn't work (maybe the interaction between the two Prism instances?)

Reply:
Indeed the highlighting doesn't seem to work at all in our website – still under investigation